spec/intl.html

```diff
@@ -701,17 +701,17 @@ <h1>
           1. Let _bestFormat_ be DateTimeStyleFormat(_dateStyle_, _timeStyle_, _styles_).
           1. <ins>If _dateStyle_ is not *undefined*, then</ins>
             1. <ins>Set _dateTimeFormat_.[[TemporalPlainDateFormat]] to _bestFormat_.</ins>
-            1. <ins>Set _dateTimeFormat_.[[TemporalPlainYearMonthFormat]] to _bestFormat_.</ins>
-            1. <ins>Set _dateTimeFormat_.[[TemporalPlainMonthDayFormat]] to _bestFormat_.</ins>
+            1. <ins>Set _dateTimeFormat_.[[TemporalPlainYearMonthFormat]] to GetDateTimeFormat(_styles_, _formatMatcher_, _bestFormat_, ~year-month~, ~year-month~, ~relevant~).</ins>
```

Comment:
Thanks. For the moment, I've updated this PR with a prose description of what should happen. I'll see if that can be improved.

Reply:
@anba I've updated the PR again with a better description of this. It is similar to what you suggested, except that I don't think we really need to call GetDateTimeFormat since we know whether relevant options are present or not, and we don't need defaults. Instead we can just pass the options directly to the format matcher. Please let me know if this works for you.